modules/layers/src/geojson-layer/geojson-layer.js

```diff
@@ -157,13 +159,90 @@ export default class GeoJsonLayer extends CompositeLayer {
     const LineStringsLayer = this.getSubLayerClass('line-strings', PathLayer);
     const PointsLayer = this.getSubLayerClass('points', ScatterplotLayer);
 
+    const layerProps = {
+      points: {data: undefined, _dataDiff: undefined},
+      lines: {data: undefined, _dataDiff: undefined},
+      polygons: {data: undefined, _dataDiff: undefined},
+      polygonsOutline: {data: undefined, _dataDiff: undefined}
```

Comment:
Why initialize a prop as `undefined`?

Reply:
Just to try to explain the structure the object, but open to remove it

Reply:
I'm going to add the structure just like a comment